utils/drop.ts

```diff
@@ -1,5 +1,13 @@
 export const DEFAULT_DROP = '/ahp/drops/chroma'
 
+export const HOLDER_OF_DROP_MAP = {
+  '50': '51',
+}
+
```

Comment:
This is obsolete then ?

Reply:
it's used to check if the "exclusive drop" is live and can mint , makes no sense to show that message if the drop has ended